src/components/HomePage/SearchArea/useUrlSearchParams.spec.js

```diff
@@ -83,10 +84,15 @@ describe("useUrlSearchParams", () => {
                 "jobMinDuration": minDuration,
                 "jobMaxDuration": maxDuration,
             };
-        } else if (fieldName === "techs")
+        } else if (fieldName === "techs") {
             params = {
                 "technologies": expectedValue,
             };
+        } else if (fieldName === "showHidden") {
+            params = {
+                "showHidden": expectedValue,
+            };
+        }
```

Comment:
This is not needed here as `showHidden` already appears on the params object: these transformations are here in case the field has a different name than expected on the value itself cannot be directly present for some reason; that is not the case with `showHidden`.